operators/online_video_reasoner/tests/test_online_video_reasoner.py

```
@@ -0,0 +1,685 @@
+            for text in ("A vehicle ", "moves left."):
+                event = {"choices": [{"delta": {"content": text}}]}
+                self.wfile.write(f"data: {json.dumps(event)}\n\n".encode())
+                self.wfile.flush()
+            self.wfile.write(b"data: [DONE]\n\n")
+        else:
+            self.send_header("Content-Type", "application/json")
+            self.end_headers()
+            response = {"choices": [{"message": {"content": "A static scene."}}]}
+            self.wfile.write(json.dumps(response).encode())
+
+    def log_message(self, format, *args):
```

Ruff (0.15.21)

[error] Function argument `format` is shadowing a Python builtin

(A002)